README.md

```diff
@@ -1,23 +1,58 @@
-# MetOfficeAMD
+# MetOfficeDataHub
 Python wrapper around MetOffice Atmospheric Model Data REST API
 
 [![codecov](https://codecov.io/gh/openclimatefix/MetOfficeAMD/branch/main/graph/badge.svg?token=64JOBKZNCI)](https://codecov.io/gh/openclimatefix/MetOfficeAMD)
 
-IN DEVELOPMENT
+MetOfficeAMD is a simple wrapper for the API provided by the British Met Office
+<https://metoffice.apiconnect.ibmcloud.com/metoffice/production/>_ known as Weather DataHub.
+It can be used to retrieve weather observations and forecasts. The aim is to focus on the AMD -
+Atmosphere Model Data REST API
+<https://metoffice.apiconnect.ibmcloud.com/metoffice/production/product/17502/api/16908>
 
-MetOfficeAMD is a simple wrapper for the API provided by the British Met Office <https://metoffice.apiconnect.ibmcloud.com/metoffice/production/>_ known as Weather DataHub. It can be used to retrieve weather observations and forecasts. The aim is to forecus on the AMD - Atmosphere Model Data REST API <https://metoffice.apiconnect.ibmcloud.com/metoffice/production/product/17502/api/16908>
+This currently download only the latest results, i.e no historic data can be fetched.
+The timestamp can be seen in the grib / xarray files.
+
+This (unofficial) library has been built and is maintained by OCF, not the MetOffice.
 
 ## Access
-Your need to register and obtain a API key and secret. This should be placed in environment variables as `API_KEY` and `API_SECRET`
+Your need to register and obtain an API key and secret. This should be placed in environment variables
+as `API_KEY` and `API_SECRET`
+
+## Python
+
+### Installation
+
+Install directly from pypi using
+```pip install metofficedatahub```
+
+### Example
+
+```python
+from metofficedatahub.multiple_files import MetOfficeDataHub
+
+# 1. Get data from API, download grib files
+datahub = MetOfficeDataHub(client_id="fake", client_secret="fake")
+datahub.download_all_files(order_ids=["test_order_id"])
+
+# 2. load grib files to one Xarray Dataset
+data = datahub.load_all_files()
+```
+
+### Application
 
-## Installation
+You can run it directly with python using
+```bash
+python3 metofficedatahub/app.py --save-dir="s3://bucket/folder"
+```
+which will download all the files from NWP, join them together into a xarray dataset, and then save them.
```

Comment:
`from NWP`

will it not download from the Weather DataHub?

Reply:
And thanks, yea good catch